kit/transport/http/error_handler.go

```diff
@@ -35,7 +35,8 @@ func (h ErrorHandler) HandleHTTPError(ctx context.Context, err error, w http.Res
 
 	code := errors2.ErrorCode(err)
 	var msg string
-	if _, ok := err.(*errors2.Error); ok {
+	var eTmp *errors2.Error
+	if ok := errors.As(err, &eTmp); ok {
```

Comment:
Why `As` with a tmp var instead of `Is`? I noticed this pattern in a few places, so I assume its intentional but I'm not sure why.

Reply:
For `errors.Is` you have to provide a concrete object that is matched against for equality. In `errors.As` you are trying to find something that is assignable to the target. Since the influx `Error` struct actually can represent tons of errors, assignable is all we really want. I don't want field-by-field comparison.